.buildkite/pipelines/quality-gates/pipeline.tests-production-canary.yaml

```diff
@@ -0,0 +1,15 @@
+# These pipeline steps constitute the quality gate for your service within the production-canary environment.
+# Incorporate any necessary additional logic to validate the service's integrity.
+# A failure in this pipeline build will prevent further progression to the subsequent stage.
+
+steps:
+  - label: ":pipeline::rocket::seedling: Trigger control-plane e2e tests"
+    trigger: "ess-k8s-production-e2e-tests" # https://buildkite.com/elastic/ess-k8s-production-e2e-tests
+    build:
+      env:
+        REGION_ID: aws-us-east-1
+        NAME_PREFIX: ci_test_kibana-promotion_
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-production-canary.yaml)"
+
+  - label: ":cookie: 24h bake time before continuing promotion"
+    command: "sleep 86400"
```

Comment:
I was weighing whether we should try to start with a manual verification instead of 24h timer, but I think the 24h timer as you've done here is for the best (and is the direction we should be moving anyway). Since _ideally_ we have enough confidence by the time we get to production canary that we'd be alerted if there are any issues once the full rollout happens (plus we are running e2e tests again).

It just means whoever does the final release to prod canary needs to be thinking ahead and aware that the full rollout will be happening ~24h later. (Even more reason to make sure we aren't doing a prod canary release the day before a weekend or holiday)